ogr/abstract.py

```diff
@@ -1335,7 +1341,7 @@ def __init__(self, repo: str, service: GitService, namespace: str) -> None:
 
                 - GitHub: username or org name.
                 - GitLab: username or org name.
-                - Pagure: namespace (e.g. `"rpms"`).
+                - Pagure: namespace (e.g. `"rpms"`). May be `None`, i.e. no namespace present.
```

Comment:
What about using an empty string for this?

(I am looking at that from the `GitLab` perspecive where you can have `"one_namespace"`, `"two/namespaces"`, so why not `""`.)

I know, it's a functional change...;)